features/dd-sdk-android-profiling/src/main/java/com/datadog/android/profiling/internal/ProfilingFeature.kt

```diff
@@ -67,6 +72,7 @@ internal class ProfilingFeature(
             )
             return
         }
+        this.ttidEvent = event
         profiler.stop()
```

Comment:
minor: for the future, since we may have different profile starts reasons, it may be better to simply pass `ttidEvent` as an argument of `stop` (obviously, it will have multiple overloads).

And then, inside the stop we can bundle result with event by checking the result tag and pass both in `ProfilingSuccess`, this allows better result bundling.

not blocking for now.